kv/memberlist/memberlist_client.go

```diff
@@ -428,7 +442,12 @@ func (m *KV) starting(_ context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to create memberlist: %v", err)
 	}
-
+	// Spawn notify workers.
+	numNotifyWorkers := runtime.NumCPU()
+	for i := 0; i < numNotifyWorkers; i++ {
+		m.notifyWorkersWG.Add(1)
```

Comment:
```suggestion
	m.notifyWorkersWG.Add(numNotifyWorkers)
	for i := 0; i < numNotifyWorkers; i++ {
```

Reply:
resolving, as with new per-key goroutine implementation does not longer apply.